Controller/Component/Auth/ApiAuthenticate.php

```diff
@@ -49,6 +49,7 @@ public function getUser(CakeRequest $request) {
 		try {
 			$jwt = (new ApiToken())->decode($this->_getAuthString($request), $this->_secretKey);
 		} catch (Exception $e) {
+			throw $e;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the code after this "throw". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS1763)